Tests/GRPCCodeGenTests/Internal/Translator/ClientCodeTranslatorSnippetBasedTests.swift

```diff
@@ -110,17 +111,17 @@ final class ClientCodeTranslatorSnippetBasedTests: XCTestCase {
     let expectedSwift =
       """
       /// Documentation for ServiceA
-      protocol namespaceA_ServiceAClientProtocol: Sendable {
+      public protocol namespaceA_ServiceAClientProtocol: Sendable {
           /// Documentation for MethodA
-          func methodA<R>(
+          public func methodA<R>(
```

Comment:
Methods in `protocol` _declarations_ don't have any access level: the level is dictated by the level on the protocol.